compiler/rustc_resolve/src/late.rs

```diff
@@ -4123,6 +4126,7 @@ impl<'a: 'ast, 'b, 'ast, 'tcx> LateResolutionVisitor<'a, 'b, 'ast, 'tcx> {
                     && PrimTy::from_name(path[0].ident.name).is_some() =>
             {
                 let prim = PrimTy::from_name(path[0].ident.name).unwrap();
+                self.check_prim_gate(prim, path[0].ident.span);
```

Comment:
Please inline this now since it's only called once.

Reply:
Done